PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 ## Essential Checklist
 
 - [ ] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...", followed by a short, clear summary of the changes.
-- [ ] I have followed the build process and error check mentioned in the [readme file](https://github.com/oppia/guppy/blob/master/README.md).
+- [ ] I have followed the build process and verifying your changes mentioned in the [readme file](https://github.com/oppia/guppy/blob/master/README.md).
```

Comment:
I have followed the build process and "verifying your changes" section mentioned in the ...

Reply:
Done!